### ISSUE TITLE
Schema Indexes and Constraints

### ISSUE THREAD

**Issue by jadell** [jadell/neo4jphp#98]
http://docs.neo4j.org/chunked/2.0.0-M06/rest-api-schema-indexes.html
http://docs.neo4j.org/chunked/2.0.0-M06/rest-api-schema-constraints.html

**Comment by bzier** [comment]
Any progress on this @jadell? Any idea if anyone is working it or how close it is?

**Comment by jadell** [comment]
No progress, and there probably won't be. Sorry. Have you looked at https://github.com/graphaware/neo4j-php-client ?

**Comment by bzier** [comment]
Yeah, that's what I figured given how much time had passed, but thought it was worth asking. I did see the other graphaware library you linked to, but I'm playing with this package at the moment: https://github.com/lphuberdeau/Neo4j-PHP-OGM
and it is using yours as the underlying library. Rewriting that to use the graphaware library, or including support for the schema components in yours are both somewhat significant undertakings. I'll have to take a closer look at each and see which is more feasible. Thanks for the reply.
